src/field_theory/splitting_field/test.lean

```diff
@@ -0,0 +1,5 @@
+import field_theory.splitting_field.construction
+
+structure test (K : Type) [inst : field K] :=
```

Comment:
This causes a timeout that I really don't understand. Does anyone has an idea?

Reply:
I posted some further minimisations at https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/!4.234891.20.20FieldTheory.2ESplittingField/near/365719262, but don't have a solution.